meson_options.txt

```diff
@@ -191,98 +131,171 @@ option(
     description: 'Use OS specific init config',
 )
 option(
-    'disable-init-hooks',
+    'with-init-hooks',
     type: 'boolean',
-    value: false,
-    description: 'Disable install hooks for OS specific init system',
+    value: true,
+    description: 'Enable install hooks for OS specific init system',
 )
 option(
     'with-kerberos',
-    type: 'feature',
-    value: 'auto',
-    description: 'Kerberos V support',
+    type: 'boolean',
+    value: true,
+    description: 'Enable Kerberos V API support',
+)
+option(
+    'with-krbV-uam',
+    type: 'boolean',
+    value: true,
+    description: 'Enable build of Kerberos V UAM module',
```

Comment:
Is there a scenario when with-kerberos and with-krbV-uam are enabled separately? In other words, would you ever enable the Kerberos V API without also enabling the Kerberos V UAM?